verifier/src/runner.rs

```diff
@@ -353,7 +353,7 @@ impl VerificationRunner {
             }
             score_entries_map
         };
-        convergence_check(lt.clone(), seed, &score_entries).map_err(Error::Algo)
+        convergence_check(lt.clone(), seed.clone(), &score_entries).map_err(Error::Algo)
```

Comment:
Nice catch, would've jeopardized seed update after initial use.

Reply:
No, we didn't, we just passed the reference, but now we pass the value by cloning it